street-explorer/js/main.js

```diff
@@ -133,13 +133,14 @@ class TestCase {
     const geometry = await loadFile(prefix + "geometry.json");
     const boundary = JSON.parse(await loadFile(prefix + "boundary.json"));
 
-    const geometryLayer = makePlainGeoJsonLayer(geometry);
+    // TODO The geometry layer is vastly simplified from makePlainGeoJsonLayer, because we don't have the network to plumb in.
```

Comment:
When you first load a test from a file, the "geometry" layer is totally minimal and not clickable now:
![Screenshot from 2023-03-10 16-51-38](https://user-images.githubusercontent.com/1664407/224375231-cdd23034-2b06-4658-915a-13fa8ec6d8f6.png)
I couldn't find an easy way to plumb around all the stuff we need for the more detailed clickable intersection polygon layer.